Release/include/cpprest/details/SafeInt3.hpp

```diff
@@ -6721,13 +6723,13 @@ class SafeInt
     // Miscellaneous helper functions
     SafeInt<T, E> Min(SafeInt<T, E> test, const T floor = IntTraits<T>::minInt) const SAFEINT_NOTHROW
     {
-        T tmp = test < m_int ? (T)test : m_int;
+        T tmp = (T)test < m_int ? (T)test : m_int;
```

Comment:
Again this gives up safety. You can punch all the way down to GreaterThanTest et al. if you wish but just doing the comparison in the T domain is not safe.